section2/exercises/conditionals.rb

```diff
@@ -0,0 +1,14 @@
+#This work was all done in IRB, but I typed my commands in atom
+# for your reference.
+
+def water_status(minutes)
```

Comment:
This is great! You used the Ruby conventions for indentation, whitespace, spaces, etc. which makes your code very readable💯